src/windows/video_widget.py

```diff
@@ -286,12 +298,12 @@ def update_title(self):
 
         if self.settings.get("preview-fps"):
             # Update window title with FPS output
-            dock.setWindowTitle(_("Video Preview") + " " + _("(Paint: %d FPS, Render: %d FPS, %dx%d)")
+            dock.setWindowTitle(_("Video Preview") + speed_str + _("(Paint: %d FPS, Render: %d FPS, %dx%d)")
                                                       % (self.paint_fps, self.present_fps,
                                                          rect.width() * scale, rect.height() * scale))
         else:
             # Restore window title
-            dock.setWindowTitle(_("Video Preview"))
+            dock.setWindowTitle(_("Video Preview") + speed_str)
```

Comment:
I'm thinking we keep this in parenthesis, similar to the FPS above: `Video Preview (2x)`. Also, let's omit `0x` and `+-1x` speeds (since they are just noise). I mostly want to display the **speed** when it's not in a normal speed.

For example:
 - SPEED 0:  Video Preview
 - SPEED 1:  Video Preview
 - SPEED 2:  Video Preview (2x)
 - SPEED 4:  Video Preview (4x)
 - SPEED -1:  Video Preview
 - SPEED -2:  Video Preview (-2x)
 - SPEED -4:  Video Preview (-4x)